<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>603</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>872</width>
    <height>603</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CESI-Project</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="customerTab">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="title">
          <string>Customer Tab</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTableWidget" name="customerListTable">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <property name="gridStyle">
               <enum>Qt::SolidLine</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="verticalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>firstname</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>lastname</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>birth_date</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>address_line</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>postal_code</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>city</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="">
            <property name="minimumSize">
             <size>
              <width>395</width>
              <height>471</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>395</width>
              <height>471</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <item alignment="Qt::AlignHCenter|Qt::AlignTop">
              <widget class="QLabel" name="titleCustomer">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="accessibleName">
                <string>titleCustomer</string>
               </property>
               <property name="text">
                <string>Add Customer Form</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>First Name</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="customerFormFirstName">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Last Name</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="customerFormLastName"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Birth Date</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="customerFormBirthDate">
                 <property name="acceptDrops">
                  <bool>false</bool>
                 </property>
                 <property name="accelerated">
                  <bool>false</bool>
                 </property>
                 <property name="currentSection">
                  <enum>QDateTimeEdit::DaySection</enum>
                 </property>
                 <property name="calendarPopup">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="font">
                    <font>
                     <family>MS Shell Dlg 2</family>
                     <pointsize>10</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Delivery Address</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_18">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>Address Line</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="customerFormAddressLine1"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_15">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_16">
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>City</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="customerFormCity1"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_17">
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Postal code</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="customerFormPostalCode1">
                       <property name="minimum">
                        <number>10000</number>
                       </property>
                       <property name="maximum">
                        <number>99999</number>
                       </property>
                       <property name="stepType">
                        <enum>QAbstractSpinBox::DefaultStepType</enum>
                       </property>
                       <property name="value">
                        <number>10000</number>
                       </property>
                       <property name="displayIntegerBase">
                        <number>10</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="font">
                    <font>
                     <family>MS Shell Dlg 2</family>
                     <pointsize>10</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Billing Address</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_18">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>Address Line</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="customerFormAddressLine2"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_15">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_16">
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>City</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="customerFormCity2"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_17">
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Postal code</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="customerFormPostalCode2">
                       <property name="minimum">
                        <number>10000</number>
                       </property>
                       <property name="maximum">
                        <number>99999</number>
                       </property>
                       <property name="stepType">
                        <enum>QAbstractSpinBox::DefaultStepType</enum>
                       </property>
                       <property name="value">
                        <number>10000</number>
                       </property>
                       <property name="displayIntegerBase">
                        <number>10</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QPushButton" name="pushButtonAddCustomer">
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="accessibleName">
                  <string>pushButtonAddCustomer</string>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Add Customer</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="orderTab">
         <attribute name="title">
          <string>Order Tab</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="staffTab">
         <attribute name="title">
          <string>Staff Tab</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="staffTabFirstNameInput">
              <property name="text">
               <string>rtrthrthrth</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
